.gitignore

```diff
@@ -99,3 +99,11 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# Vim leftover
```

Comment:
Not everyone uses vim, and this is not strictly related to the schema. Maybe put this in your private ignore list? (.git/info/exclude)

Reply:
Sure, can do. I normally see these left in the `.gitignore` files, is there a reason to move them out?

Reply:
IMO projects should have just entries relevant to their code. But of course it's not super important.